es6/jsxcad-api-shape.js

```diff
@@ -1297,6 +1297,19 @@ const clipFrom = Shape.registerMethod3(
   clipFrom$1
 );
 
+const cloud = Shape.registerMethod3(
```

Comment:
Is cloud a point cloud?

Reply:
Yes, just a set of points sampled at a given resolution.